testing/cucumber/features/linters/test_with_action_as_final_step.feature

```diff
@@ -22,7 +22,28 @@ Feature: Test with action step as final step linter
       | linter                              | problem                            | location         |
       | TestWithActionStepAsFinalStepLinter | Test has 'When' as the final step. | <path_to_file>:3 |
 
-  @wip
-  Scenario: Configuration
+  Scenario: Configuration of keywords for different dialect
+    Given a linter for tests with an action step as the final step has been registered
+    And the following configuration file:
+      """
+      TestWithActionStepAsFinalStepLinter:
+        Given:
+          - Dado
+        When:
+          - Quando
+        Then:
+          - Então
+      """
+    And the following feature:
+      """
+      # language:pt
+      Funcionalidade: Feature name
 
-  Configure the keyword(s) that count as action steps?
+        Cenário: scenario name
+          Quando this is an action in pt dialect
+      """
+    When the configuration file is loaded
+    And it is linted
+    Then an error is reported:
+      | linter                              | problem                            | location         |
+      | TestWithActionStepAsFinalStepLinter | Test has 'When' as the final step. | <path_to_file>:4 |
```

Comment:
Same deal here and for the other feature files that demonstrate configuration. Please use a config file that has both single keyword synonyms and multiple keyword synonyms so that users won't have to second guess themselves on if it is possible or not.